test/test_fx_experimental.py

```diff
@@ -749,6 +751,97 @@ def test_subgraph_trivial_resnet(self):
         module_with_submodules = split_module(traced, m, lambda node: 0)
         module_with_submodules(a)
 
+    @skipIfNoTorchVision
+    def test_normalize_args(self):
+        m = resnet18()
+
+        class FunctionalTracer(torch.fx.Tracer):
+            def is_leaf_module(self, m: torch.nn.Module, module_qualified_name : str) -> bool:
+                leaves = set([torch.nn.BatchNorm2d])
+                return type(m) in leaves
+
+        traced = torch.fx.GraphModule(m, FunctionalTracer().trace(m))
+
+        input = torch.randn(5, 3, 224, 224)
+        ref_outs = traced(input)
+
+        traced = NormalizeArgs(traced).transform()
+
+        test_outs = traced(input)
+        self.assertEqual(test_outs, ref_outs)
+
+        modules = dict(traced.named_modules())
+        for node in traced.graph.nodes:
+            if node.op == 'call_function' and node.target.__module__ == 'torch.nn.functional':
+                self.assertEqual(len(node.args), 0)
+            if node.op == 'call_module':
+                submod_class = modules[node.target].__class__
+                nn_class = getattr(torch.nn, submod_class.__name__)
+                if submod_class == nn_class:
+                    self.assertEqual(len(node.args), 0)
+
+    def test_normalize_modules_exhaustive(self):
+        """
+        Exhaustively test `NormalizeArgs` on all standard
+        torch.nn Module classes
+        """
+        for test_params in module_tests + new_module_tests:
+            if 'constructor' not in test_params:
```

Comment:
I assume these are special case things, might be worth adding a brief comment to explain why.

Reply:
Honestly not sure why this is the way it is. This is copypasta from `torch_nn`: https://github.com/pytorch/pytorch/blob/b2d8f0a431f8ce2c606ed25ac4d28287489c71bc/test/test_nn.py#L10063

What i'm trying to do here is just get instances + example inputs for all the stdlib `nn.Module` instances